projects/objects/school_furniture/protos/Clock.proto

```diff
@@ -14,6 +14,7 @@ PROTO Clock [
     rotation IS rotation
     children [
       DEF MINUTE_HAND Transform {
+        rotation 0.5774 0.5773 0.5773 2.0944
```

Comment:
Won't be better to rate the IFS coordinates instead?
Same comment for all the other IFS in this file.